restapi/endpoints_test.go

```diff
@@ -242,7 +229,7 @@ func (suite *endpointTestSuite) TestCreateAuthorFailsWithUnknownCountry() {
 		},
 	}
 
-	response := suite.endpoint.POST("/authors").WithHeader("Authorization", fmt.Sprintf("Bearer %s", suite.token)).
+	response := suite.endpoint.POST("/authors").
```

Comment:
Wait the endpoint doesn't need a header anymore?

Reply:
No, I moved the endpoints around so this is no longer protected (temporarily) and the tests are updated accordingly so they don't fail (the tests will be all updated once at the end of this auth epic)